Mbody_example.py

```diff
@@ -29,47 +19,44 @@
 bu.insert_general_benchmark_code(config)
 
 # Number of neurons
-N_AL = 100
-N_MB = int(2500*config['scale'])
-N_LB = 100
+N_PN = 100
+N_iKC = int(2500*config['scale'])
+N_eKC = 100
 # Constants
 g_Na = 7.15*uS
-E_Na = 50*mV
-g_K = 1.43*uS
-E_K = -95*mV
-g_leak = 0.0267*uS
-E_leak = -63.56*mV
-C = 0.3*nF
+V_Na = 50*mV
+g_Kd = 1.43*uS
+V_Kd = -95*mV
+g_L = 0.0267*uS
+V_L = -63.56*mV
+C_M = 0.3*nF
 VT = -63*mV
-# Those two constants are dummy constants, only used when populations only have
-# either inhibitory or excitatory inputs
-E_e = 0*mV
-E_i = -92*mV
-# Actual constants used for synapses
-NKCKC= N_MB
-if NKCKC > 10000:
-    NKCKC = 10000
-g_scaling = NKCKC/2500
```

Comment:
The problem would be whether this does an integer division (would by a pure Python problem).

Reply:
... but if you tested in Python 3 and it's not an integer division in Python 3 it should be fine...

Reply:
But to be 100% sure, could you revert to the g_scaling variant and test?